xbmc/FileItem.cpp

```diff
@@ -675,7 +675,7 @@ void CFileItem::ToSortable(SortItem &sortable, Field field) const
 void CFileItem::ToSortable(SortItem &sortable, const Fields &fields) const
 {
   Fields::const_iterator it;
-  for (it = fields.begin(); it != fields.end(); it++)
+  for (it = fields.begin(); it != fields.end(); ++it)
```

Comment:
That would be really nice but we can't demand this. I would accept as is because it is already an improvement.